js/core/stdio/stdout.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014-2015 runtime.js project authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+class stdout {
+  constructor() {
+
+  }
+
+  write(text, fg, bg) {
+    runtime.tty.print(text, 1, fg, bg);
+  }
+
+  writeln(text, fg, bg) {
```

Comment:
`onwrite` by default is not connected to a screen, only `defaultStdout` instance outputs written data to the screen
```js
defaultStdout.onwrite = function(data) { runtime.tty.print(data, fg, bg); };
```
myStdout (as in example above) could do a completely different thing. You could even pipe data from one stdout to another like
```js
myStdout.onwrite = otherStdout.onwrite;
```